fashrevwall/run_streaming.py

```diff
@@ -0,0 +1,8 @@
+from TwitterClient import TwitterClient
```

Comment:
I will need to read a bit more about the streaming  + heroku, we might need to use a heroku worker if we want it constantly listening.